packages/flutter_tools/test/commands.shard/hermetic/build_ipa_test.dart

```diff
@@ -46,6 +46,15 @@ class FakeXcodeProjectInterpreterWithBuildSettings extends FakeXcodeProjectInter
       'DEVELOPMENT_TEAM': 'abc',
     };
   }
+
+  Version _version = Version(14, 0, 0);
```

Comment:
I think it would be cleaner to add to the constructor:
```dart
  FakeXcodeProjectInterpreterWithBuildSettings({
    this.overrides = const <String, String>{},
    Version? version,
  }) : version = version ?? Version(14, 0, 0);
...
  @override
  final Version version;
```

Reply:
done